.github/workflows/main.yml

```diff
@@ -51,24 +51,41 @@ jobs:
                # Strip "v" prefix from tag name
                [[ "${{ github.ref }}" == "refs/tags/"* ]] && IMAGE_VERSION=$(echo $IMAGE_VERSION | sed -e 's/^v//')
                echo "IMAGE_VERSION=$IMAGE_VERSION" >> $GITHUB_ENV
-
+         
                echo "IMAGE_ID=$(echo ghcr.io/${{ github.repository_owner }}/$IMAGE_NAME | tr '[A-Z]' '[a-z]')" >> $GITHUB_ENV
 
-         # region See: https://docs.github.com/en/packages/managing-github-packages-using-github-actions-workflows/publishing-and-installing-a-package-with-github-actions#upgrading-a-workflow-that-accesses-ghcrio
-         -  name: Build Image
-            run: |
-               docker build -t $IMAGE_NAME .
+         # region See: https://docs.github.com/en/packages/managing-github-packages-using-github-actions-workflows/publishing-and-installing-a-package-with-github-actions#upgrading-a-workflow-that-accesses-ghcrio     
+         -  name: Use QEMU
+            uses: docker/setup-qemu-action@master
+            with:
+             platforms: all
+                      
+         - name: Free Disk Space (Ubuntu)
```

Comment:
why do we need this?

if definitely necessary, should this be pulled higher up in the step order?

Reply:
Because, otherwise github action runner will run out of space throughout my extensive testing, github puts a lot of tools that we do not need inside the runner. Will push it to the top

Reply:
during _pushing_ sounds a little odd; can you temporarily remove this step, and post a link to the resulting action run?

Reply:
Yeah, will do. let me comment it out

Reply:
https://github.com/kokomo123/5etools-img/actions/runs/10560354113 Here you are, it just fails with no space left on device.

Reply:
cheers; if you can stick a comment e.g.

```
# Avoid the following error when building multiple large images:
# `System.IO.IOException: No space left on device : '/home/runner/runners/2.319.1/_diag/Worker_20240826-130726-utc.log'`
# See also: https://docs.github.com/en/actions/using-github-hosted-runners/using-github-hosted-runners/about-github-hosted-runners#standard-github-hosted-runners-for-public-repositories
```

above the step, that'll do just fine

---

I'd also get rid of the `tools-cache: true` line (+comments), as:

> ... removes all the pre-cached tools (Node, Go, Python, Ruby, ...) that are loaded in a runner's environment ...

and I'd rather leave anything Node alone, as those may actually get used, and `tools-cache` is not an obvious name for later debugging

Reply:
Noted, pushing fix now.